### DISCUSSION TITLE
advice on new installation

### DISCUSSION THREAD

**Discussion by IoSonoAndreaZ** [immich-app/immich#4744] [Q&A] [answered]
Hi, i'm coming from Google Photos, went through Plex and now trying to setup Immich for my needings.
On my computer I have many GB of photos, half of them also on my phone.
My desire is to rasily import everything already on my computer in Immich, and start to backup from my phone every photo taken FROM NOW ON.
I played a bit and added the plex directory to Immich (read only), everything seems to work fine.
Now I want to setup my phone to backup from today and not all the old photos already on my computer.
How to achieve this?
Thanks
Andrea

**Comment by bo0tzz** [comment, marked as answer]
If the files are identical, Immich should not upload any files that are already on the server.
